### DISCUSSION TITLE
Querying a part of collection

### DISCUSSION THREAD

**Discussion by bhavya-giri** [milvus-io/milvus#32557] [Q&A and General discussion]
Is there a way to specifically query with specific nodes in collection, say for example: I indexed two book harry porter and lotr in a collection_name="book". 
There is metadata for both the books - 1) file_name:"harry_porter.txt" , 2) file_name:"lotr.txt"

Is there way i give a parameter i just want query with file_name:"lotr.txt"? Maybe using Langchain or LlamaIndex

**Comment by xiaofan-luan** [comment]
filename should be a filed, and you should use expression file_name="lotr.txt".
Read the document and you should understand those
